code/__DEFINES/preferences.dm

```diff
@@ -19,12 +19,13 @@
 #define CHAT_DEBUGLOGS	128
 #define CHAT_LOOC		256
 #define CHAT_GHOSTRADIO 512
-#define SHOW_TYPING 	1024
-#define DISABLE_SCOREBOARD 2048
-#define DISABLE_KARMA_REMINDER	4096
-#define MEMBER_PUBLIC	8192
-#define CHAT_NO_ADMINLOGS 16384
-#define DONATOR_PUBLIC	32768
+#define CHAT_GHOSTANONSAY 1024
+#define SHOW_TYPING 	2048
+#define DISABLE_SCOREBOARD 4096
+#define DISABLE_KARMA_REMINDER	8192
+#define MEMBER_PUBLIC	16384
+#define CHAT_NO_ADMINLOGS 32768
+#define DONATOR_PUBLIC	65536
```

Comment:
You simply can't use `toggles` to store another setting. Every available bit is used.

You'll probably have to add yet another column to the SQL.